xemc3/core/load.py

```diff
@@ -1519,6 +1519,68 @@ def write_info_file(fn: str, ds: xr.Dataset) -> None:
             "TOTAL_RAD": dict(long_name="Total radiation", units="W"),
         },
     ),
+    "ENERGY_TRANSPORT": dict(
+        type="mapping",
+        vars={
+            "qconve": dict(
+                units="Wm^{-2}", long_name="Parallel electron convective heat flux"
+            ),
+            "qconvi": dict(
+                units="Wm^{-2}", long_name="Parallel ion convective heat flux"
+            ),
+            "qconde": dict(
+                units="Wm^{-2}", long_name="Parallel electron conductive heat flux"
+            ),
+            "qcondi": dict(
+                units="Wm^{-2}", long_name="Parallel ion conductive heat flux"
+            ),
+        },
+    ),
+    "ENERGY_BALANCE": dict(
+        type="mapping",
+        vars={
+            "div_qconve": dict(
+                units="10^{-6}Wm^{-3}",
```

Comment:
```suggestion
                units="Wm^{-3}",
                scale=1e6,
```
Similar for the other ones.